app/config/collections.php

```diff
@@ -485,6 +485,147 @@
         ],
     ],
 
+    'vcs' => [
+        '$collection' => ID::custom(Database::METADATA),
+        '$id' => ID::custom('vcs'),
+        'name' => 'vcs',
+        'attributes' => [
+            [
+                '$id' => ID::custom('installationId'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('projectId'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('projectInternalId'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('provider'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('accessToken'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16384,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => ['encrypt']
+            ]
+        ],
+        'indexes' => [
+        ],
+    ],
+
+    'vcs_map' => [
```

Comment:
We might need to start adding a small description for every collection. @vermakhushboo we can do it on all collections here, a bit tedious, but will be a good practice in getting more familiar with the entire architecture.